src/components/RelicModal.tsx

```diff
@@ -173,20 +200,27 @@ export default function RelicModal(props: {
       relic.verified = false
     }
 
-    console.log('Completed relic', relic)
+    console.log(t('relic.messages.editsuccess'), relic)
 
     props.onOk(relic)
     props.setOpen(false)
   }
   const onFinishFailed = () => {
-    Message.error('Submit failed!')
+    Message.error(t('relic.messages.submitfail'))
     props.setOpen(false)
   }
   const onValuesChange = (formValues: RelicForm) => {
     let mainStatOptions: MainStatOption[] = []
     if (formValues.part) {
       mainStatOptions = Object.entries(Constants.PartsMainStats[formValues.part]).map((entry) => ({
-        label: entry[1],
+        label: (() => {
+          return (
+            <Flex align='center' gap={10}>
+              <img src={Assets.getStatIcon(entry[1], true)} style={{ width: 22, height: 22 }}/>
+              {t(`common:stats.${entry[1]}`)}
+            </Flex>
+          )
+        })(),
```

Comment:
Why is this change needed? It looks like its defining an inline function then just calling it immediately?